services/galley/schema/src/Main.hs

```diff
@@ -99,7 +100,8 @@ main = do
       V51_FeatureFileSharing.migration,
       V52_FeatureConferenceCalling.migration,
       V53_AddRemoteConvStatus.migration,
-      V54_TeamFeatureSelfDeletingMessages.migration
+      V54_TeamFeatureSelfDeletingMessages.migration,
+      V55_TeamFeatureSelfDeletingMessagesPaymentStatus.migration
       -- When adding migrations here, don't forget to update
       -- 'schemaVersion' in Galley.Data
```

Comment:
You forgot to update `schemaVersion` as explained here.

Reply:
Fixed